### PR TITLE
Add missing test from nightly to r1.13 test folder.

### PR DESCRIPTION
**Pull request by vanbasten23**

### CODE REVIEW

File: tests/pytorch/r1.13/common.libsonnet [hunk 1]
```diff
@@ -24,45 +24,118 @@ local volumes = import 'templates/volumes.libsonnet';
...
+        pip install --user \
+          https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch-1.13-cp38-cp38-linux_x86_64.whl \
+          https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torchvision-1.13-cp38-cp38-linux_x86_64.whl \
+          'torch_xla[tpuvm] @ https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch_xla-1.13-cp38-cp38-linux_x86_64.whl'
         # No need to check out the PyTorch repository, but check out PT/XLA at
         # pytorch/xla anyway
         mkdir pytorch
         cd pytorch
-        git clone https://github.com/pytorch/xla.git -b r1.13
+        git clone https://github.com/pytorch/xla.git
```
Comment (JackCaoG) [inline tests/pytorch/r1.13/common.libsonnet:105]: why remove the branch?

File: tests/pytorch/r1.13/dlrm.libsonnet [hunk 2]
```diff
@@ -82,94 +83,126 @@ local utils = import 'templates/utils.libsonnet';
...
   },
+  local v4_8 = {
+    accelerator: tpus.v4_8,
+  },
   configs: [
     dlrm + v3_8 + one_core + timeouts.Hours(3) + mixins.Experimental,
     dlrm + v3_8 + seq_fwd + timeouts.Hours(3) + mixins.Experimental,
     dlrm + v3_8 + mp_fwd + timeouts.Hours(3) + mixins.Experimental,
     dlrm + v3_8 + mp_dp_fwd + timeouts.Hours(3),
-    dlrm_convergence + v3_8 + criteo_kaggle + timeouts.Hours(6),
```
Comment (JackCaoG) [inline tests/pytorch/r1.13/dlrm.libsonnet:173]: did we remove convergence here in purpose?

File: tests/pytorch/r1.13/common.libsonnet (hunk 1 above)
Comment (vanbasten23) [reply tests/pytorch/r1.13/common.libsonnet:105]: Previously I copied all test files from tests/pytorch/r1.12 and this pr is to copy them from tests/pytorch/nightly. This is how nightly version does it: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/df347546f2ff0c71ac7acbc25babdf9ef985216d/tests/pytorch/nightly/common.libsonnet#L105

File: tests/pytorch/r1.13/dlrm.libsonnet (hunk 2 above)
Comment (vanbasten23) [reply tests/pytorch/r1.13/dlrm.libsonnet:173]: hm, this convergence doesn't appear in the tests/pytorch/nightly/dlrm.libsonnet: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/df347546f2ff0c71ac7acbc25babdf9ef985216d/tests/pytorch/nightly/dlrm.libsonnet#L200-L206. Do you want me to add it?

Comment (JackCaoG) [reply tests/pytorch/r1.13/dlrm.libsonnet:173]: ok I guess we remove it from nightly for some reason, I think it is OK.

File: tests/pytorch/r1.13/common.libsonnet (hunk 1 above)
Comment (vanbasten23) [reply tests/pytorch/r1.13/common.libsonnet:105]: makes sense. Thanks.

File: tests/pytorch/nightly/common.libsonnet [hunk 3]
```diff
@@ -102,7 +102,7 @@ local volumes = import 'templates/volumes.libsonnet';
         # pytorch/xla anyway
         mkdir pytorch
         cd pytorch
-        git clone https://github.com/pytorch/xla.git
+        git clone https://github.com/pytorch/xla.git -b r1.13
```
Comment (JackCaoG) [inline tests/pytorch/nightly/common.libsonnet:105]: I think you mean to change the 1.13 file not the nightly one>

File: tests/pytorch/r1.13/common.libsonnet (hunk 1 above)
Comment (JackCaoG) [inline tests/pytorch/r1.13/common.libsonnet:105]: dito

File: tests/pytorch/nightly/common.libsonnet (hunk 3 above)
Comment (vanbasten23) [reply tests/pytorch/nightly/common.libsonnet]: good catch. thanks.

File: tests/pytorch/r1.13/common.libsonnet (hunk 1 above)
Comment (vanbasten23) [reply tests/pytorch/r1.13/common.libsonnet]: Fixed.

File: tests/pytorch/r1.13/wav2vec2.libsonnet [hunk 4]
```diff
@@ -19,6 +19,7 @@ local utils = import 'templates/utils.libsonnet';
 
 {
   local command_common = |||
+    pip install https://storage.googleapis.com/tpu-pytorch/wheels/torchaudio-nightly-cp37-cp37m-linux_x86_64.whl
```
Comment (JackCaoG) [inline tests/pytorch/r1.13/wav2vec2.libsonnet:22]: right thing to do might be actually compile and upload a 1.13 wheel, I guess the test might pass as it is.. or you can just pick a date for nightly wheel

File: tests/pytorch/r1.13/common.libsonnet [hunk 5]
```diff
@@ -24,45 +24,118 @@ local volumes = import 'templates/volumes.libsonnet';
...
+        'tpu-vm-base'
+      else
+        'tpu-vm-v4-base',
       tpuVmPytorchSetup: |||
-        sudo pip3 uninstall --yes torch torch_xla torchvision numpy
-        sudo pip3 install https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch-1.13-cp38-cp38-linux_x86_64.whl
-        sudo pip3 install https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch_xla-1.13-cp38-cp38-linux_x86_64.whl
-        sudo pip3 install https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torchvision-1.13-cp38-cp38-linux_x86_64.whl
-        sudo pip3 install https://storage.googleapis.com/cloud-tpu-tpuvm-artifacts/wheels/libtpu-nightly/libtpu_nightly-0.1.dev20220930-py3-none-any.whl
-        sudo pip3 install numpy
```
Comment (JackCaoG) [inline tests/pytorch/r1.13/common.libsonnet:50]: wondering if we will need these..

File: tests/pytorch/r1.13/wav2vec2.libsonnet (hunk 4 above)
Comment (JackCaoG) [reply tests/pytorch/r1.13/wav2vec2.libsonnet:22]: so 
```
 pip install https://storage.googleapis.com/tpu-pytorch/wheels/torchaudio+yyyymmdd-nightly-cp37-cp37m-linux_x86_64.whl
```

yyyymmdd can be when we cut the 1.13 branch

Comment (JackCaoG) [reply tests/pytorch/r1.13/wav2vec2.libsonnet:22]: test it locally to see if command actully runs.

Comment (vanbasten23) [reply tests/pytorch/r1.13/wav2vec2.libsonnet:22]: Not sure how to compile and upload a 1.13 wheel for torchaudio or if there is an instruction. Is it by `python setup.py bdist_wheel` inside pytorch/audio? And how to upload it?

As the alternative, the most recent torchaudio version is tpu-pytorch/wheels/torchaudio-nightly+20220906-cp37-cp37m-linux_x86_64.whl. Is it ok to use it?

Comment (JackCaoG) [reply tests/pytorch/r1.13/wav2vec2.libsonnet:22]: oh right.. I disabled the audio wheel build a while ago.  using 0906 make sense

File: tests/pytorch/r1.13/common.libsonnet (hunk 5 above)
Comment (vanbasten23) [reply tests/pytorch/r1.13/common.libsonnet:50]: I think they're replaced by https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/df347546f2ff0c71ac7acbc25babdf9ef985216d/tests/pytorch/nightly/common.libsonnet#L98-L100

File: tests/pytorch/r1.13/wav2vec2.libsonnet (hunk 4 above)
Comment (vanbasten23) [reply tests/pytorch/r1.13/wav2vec2.libsonnet]: replaced it.

File: tests/pytorch/r1.13/common.libsonnet (hunk 5 above)
Comment (JackCaoG) [reply tests/pytorch/r1.13/common.libsonnet:50]: sorry I meant the
```
sudo pip3 install mkl mkl-include cloud-tpu-client
        sudo apt-get -y update
        sudo apt-get install -y libomp5
```

Comment (vanbasten23) [reply tests/pytorch/r1.13/common.libsonnet:50]: I guess to validate, we'll have to run all tests? But if it fails, we'd lose one day? Do you want me to add them anyway?

**Review by JackCaoG — APPROVED** [review]

**Pull request merged**